__tests__/Tab.test.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { render } from '@testing-library/react-native';
+import { Provider } from 'react-redux';
+import { store } from '../App';
+import TabNavigation from '../src/navigations/TabNavigation/TabNavigation';
```

Comment:
Nit: Please read on how to sort these and why that might be helpful

Reply:
read it from [doc](https://eslint.org/docs/latest/rules/sort-imports#:~:text=When%20declaring%20multiple%20imports%2C%20a,purely%20a%20matter%20of%20style)

**Says that :**

When declaring multiple imports, a sorted list of import declarations make it easier for developers to read the code and find necessary imports later. This rule is purely a matter of style.